scripts/tests/check-revision-logs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export FAILURE=0
+for filename in config/sync/core.entity_form_display.node.*.default.yml
+do
+  if grep -q -A1 '\- moderation_state' ${filename}
```

Comment:
Can we use long options here for self-documenting code?
```suggestion
  if grep --quiet --after-context=1 '\- moderation_state' ${filename}
```